### DISCUSSION TITLE
Updating from 2.3.52 to 2.3.60 with big issues.

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#4692] [Unsupported Versions]
After running soup on 2.3.52 I am unable to access any tools like Alerts, Hunt, and Kibana. The version number shows 0.0.0 on SOC login page after update. Please see screenshot bottom left.

At the end of soup there was one error but looking through the log I didn't see any. Maybe a bug?

Any advice?

![image](https://user-images.githubusercontent.com/70167373/124353660-d0a62b00-dbd5-11eb-9c38-505c7432bacb.png)

**Comment by TOoSmOotH** [comment]
Not much I can help with no logs or highstate results. What does so-status say?

**Reply by ColeVan** [reply]
Everything is G2G on that end. Is there anything I need to do with so-elastic-auth or any other logs I need to look at in order to find the problem?
![image](https://user-images.githubusercontent.com/70167373/124353810-c9cbe800-dbd6-11eb-9ecc-d6414ccca8f9.png)

**Reply by TOoSmOotH** [reply]
What does the highstate say
